src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthController.java

```diff
@@ -58,12 +61,15 @@ public Health health() {
                 .build();
         }
 
-        LocalDateTime now = LocalDateTime.now(clock).minusHours(1);
-        LocalDateTime nowMinusOneHours = LocalDateTime.now().minusHours(1);
-        log.info("LocalDate Clock time minus hour {}, localDate time minus one hour {}", now, nowMinusOneHours);
+        ZoneId ukTimeZone = ZoneId.of("Europe/London");
+        ZonedDateTime utcDateTime = LocalDateTime.now(clock).atZone(ZoneOffset.UTC);
+        ZonedDateTime zonedDateTime = utcDateTime.withZoneSameInstant(ukTimeZone);
+        LocalDateTime localTime = zonedDateTime.toLocalDateTime();
 
-        if (isDateWithinWorkingHours(now)) {
-            if (repository.getNumberOfMessagesReceivedInLastHour(now) == 0) {
+        log.info("UTC time {}, localDate time {}", utcDateTime, localTime);
```

Comment:
again specify what the it's UK datetime

Reply:
I changed the variables names to the suggested ones Ravi.